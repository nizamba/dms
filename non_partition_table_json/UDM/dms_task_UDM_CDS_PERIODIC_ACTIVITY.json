{
    "rules": [
        {
            "rule-type": "transformation",
            "rule-id": "283392325",
            "rule-name": "819944968",
            "rule-target": "table",
            "object-locator": {
                "schema-name": "UDM_CDS",
                "table-name": "PERIODIC_ACTIVITY"
            },
            "parallel-load": null,
            "rule-action": "convert-lowercase",
            "value": null,
            "old-value": null
        },
        {
            "rule-type": "transformation",
            "rule-id": "450145061",
            "rule-name": "608255843",
            "rule-target": "schema",
            "object-locator": {
                "schema-name": "UDM_CDS"
            },
            "parallel-load": null,
            "rule-action": "rename",
            "value": "nizlog14_udm_cds",
            "old-value": null
        },
        {
            "rule-type": "selection",
            "rule-id": "364335004",
            "rule-name": "340464364",
            "object-locator": {
                "schema-name": "UDM_CDS",
                "table-name": "PERIODIC_ACTIVITY"
            },
            "rule-action": "include"
        }
    ]
}