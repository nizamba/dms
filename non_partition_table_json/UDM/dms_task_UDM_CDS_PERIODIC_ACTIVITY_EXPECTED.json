{
    "rules": [
        {
            "rule-type": "transformation",
            "rule-id": "409895465",
            "rule-name": "965091573",
            "rule-target": "table",
            "object-locator": {
                "schema-name": "UDM_CDS",
                "table-name": "PERIODIC_ACTIVITY_EXPECTED"
            },
            "parallel-load": null,
            "rule-action": "convert-lowercase",
            "value": null,
            "old-value": null
        },
        {
            "rule-type": "transformation",
            "rule-id": "733503967",
            "rule-name": "914851912",
            "rule-target": "schema",
            "object-locator": {
                "schema-name": "UDM_CDS"
            },
            "parallel-load": null,
            "rule-action": "rename",
            "value": "nizlog14_udm_cds",
            "old-value": null
        },
        {
            "rule-type": "selection",
            "rule-id": "929602760",
            "rule-name": "269083669",
            "object-locator": {
                "schema-name": "UDM_CDS",
                "table-name": "PERIODIC_ACTIVITY_EXPECTED"
            },
            "rule-action": "include"
        }
    ]
}